tests/factories/contacts/cht/woman.js

```diff
@@ -0,0 +1,38 @@
+const Factory = require('rosie').Factory;
+
+const parent = {
+  _id: 'clinic1',
+  parent: {
+    _id: 'hc1',
+    parent: {
+      _id: 'dist1'
+    }
+  }
+};
+
+const ephemeral_dob = {
+  dob_calendar: '2000-02-01',
+  ephemeral_months: 3,
+  ephemeral_years: 2021,
+  dob_approx: '2021-03-30',
+  dob_raw: '2000-02-01',
+  dob_iso: '2000-02-01'
+};
+
+
+Factory.define('woman')
+  .attr('_id', 'test_woman')
+  .attr('parent', parent)
+  .attr('type', 'person')
+  .attr('name', 'Mary Smith')
+  .attr('short_name', '')
+  .attr('date_of_birth_method', 'approx')
+  .attr('date_of_birth', '2000-02-01')
+  .attr('ephemeral_dob', ephemeral_dob)
```

Comment:
Are these intended to be logically linked?  ...like do you want one to be x difference than the other? If so, please do it programmatically.